scripts/energywind.lua

```diff
@@ -11,21 +11,27 @@ local UPDATE_PERIOD = 1000
 local BUILD_PERIOD = 500
 
 local turnSpeed = math.rad(20)
+local waterSpin = math.rad(30)
+local waterMaxSpin = math.rad(2)
 
 local isWind, baseWind, rangeWind
 
 local function BobTidal()
-	baseDirection = baseDirection + math.random(0,math.rad(2))
+	local spinSpeed	= 0 -- radian speed
 	while true do
-		Turn(cradle, y_axis, baseDirection, math.rad(1))
+		
+		spinSpeed = math.max(-waterMaxSpin),math.min(waterMaxSpin, 0.99*spinSpeed + (math.random() - 0.5) / 100))
```

Comment:
Looks incorrect:
```suggestion
		spinSpeed = math.max(-waterMaxSpin, math.min(waterMaxSpin, 0.99*spinSpeed + (math.random() - 0.5) / 100)))
```

But also, `0.99*spinSpeed + (math.random() - 0.5) / 100)` will never put `spinSpeed` beyond 1. Maybe it could be tweaked so that the min/max calls aren't needed (keeping in mind 1 radian is ~57 degrees).